v1.1/upgrade-cockroach-version.md

```diff
@@ -29,7 +37,7 @@ Before starting the upgrade, complete the following steps.
 
 4. [Back up the cluster](back-up-data.html). If the upgrade does not go according to plan, you can use the data to restore your cluster to its previous state.
 
-## Step 2. Perform the rolling upgrade
+## Step 3. Perform the rolling upgrade
```

Comment:
There's an on-page link to this step on line 195 that you need to change from `#step-2-perform-the-rolling-upgrade` to `#step-3-perform-the-rolling-upgrade`.

Reply:
Done.